examples/Interactions.js

```diff
@@ -0,0 +1,103 @@
+const synup = require("../dist/index")(process.env.SYNUP_KEY);
+
+const { Interactions } = synup;
+let locationId = "TWVkaWFGaWxlOjEzMDMzMg=="; 
+let respondInteraction ={
+  interactionId: "2090753a-ece6-4837-8336-8494ad308523",
+  responseContent:"This is a sample response"
+};
+
+let addInteraction ={  
+  locationId:"TG9jYXRpb246MTY4MDU=",
+  siteUrls:[  
+     {  
+        name:"trulia.com",
+        url:"test.com"
+     }
+  ]
+};
+
+let editInteraction ={  
+  "locationId":"TG9jYXRpb246MTY4MDU=",
+  "siteUrls":[  
+     {  
+        "name":"yelloyello.com",
+        "url":"https://www.yelloyello.com/places/tryvexan..."
+     },
+     {  
+        "name":"google.com",
+        "url":"https://www.google.com/search?ei=6vwBXZn..."
+     },
+     {  
+        "name":"facebook.com",
+        "url":"https://www.facebook.com/pg/thekitchenrestaurant/reviews/?ref=page_internal"
+     },
+     {  
+        "name":"maps.google.com",
+        "url":"https://www.google.com/maps/place/Foxy's+Landing..."
+     }
+  ]
+}; 
+
+
+
+
+Interactions.getInteractions(locationId)
+  .then((response) => {
+    console.log(JSON.stringify(response.data));
+    console.log("SUCCESS");
+  })
+  .catch((error) => {
+    console.log(error);
+    console.log("FAILURE");
+  });
+
+Interactions.getInteractionById()
+  .then((response) => {
+    console.log(JSON.stringify(response.data));
+    console.log("SUCCESS");
+  })
+  .catch((error) => {
+    console.log(error);
+    console.log("FAILURE");
+  });
+
+Interactions.respondToAnInteraction(respondInteraction)
+  .then((response) => {
+    console.log(JSON.stringify(response.data));
+    console.log("SUCCESS");
+  })
+  .catch((error) => {
+    console.log(error);
+    console.log("FAILURE");
+  });
+
+Interactions.getListOfInteractionSources(locationId)
```

Comment:
this function should receive 2 params, according with the service you wrote

Reply:
- So, in the service should I have to create another type to recive this kind of info and send the other parameter with this structure?
![image](https://user-images.githubusercontent.com/77854098/114622702-f0cbeb00-9c73-11eb-9fe9-16e078c53896.png)

Reply:
response type is not required, in your service file you have a function called getListOfInteractionSources and you have 2 params (locationId, params) you only need the locationId, and the other param is not required, then you must delete it from your function